x-pack/test/api_integration/apis/cloud_security_posture/update_rules_config.ts

```diff
@@ -5,83 +5,87 @@
  * 2.0.
  */
 import expect from '@kbn/expect';
-import Chance from 'chance';
-import { FtrProviderContext } from '../../ftr_provider_context';
+import type { SuperTest, Test } from 'supertest';
+import type { PackageInfo, PackagePolicy } from '@kbn/fleet-plugin/common';
+import type { CspRule } from '@kbn/cloud-security-posture-plugin/common/schemas';
+import {
+  CLOUD_SECURITY_POSTURE_PACKAGE_NAME,
+  CSP_RULE_SAVED_OBJECT_TYPE,
+  UPDATE_RULES_CONFIG_ROUTE_PATH,
+} from '@kbn/cloud-security-posture-plugin/common/constants';
+import type { SavedObjectsFindResponse } from '@kbn/core-saved-objects-api-server';
+import { packageToPackagePolicy } from '@kbn/fleet-plugin/common/services/package_to_package_policy';
+import type { FtrProviderContext } from '../../ftr_provider_context';
+
+const getNewPackagePolicy = (packageInfo: PackageInfo) =>
+  packageToPackagePolicy(
+    packageInfo,
+    'policy2', // policy2 from 'x-pack/test/functional/es_archives/fleet/agents'
+    'default',
+    'cloud_security_posture-1'
+  );
+
+const installIntegration = async (supertest: SuperTest<Test>) => {
+  const packageInfo = await getPackageInfo(supertest);
+  return await supertest
+    .post('/api/fleet/package_policies')
+    .set('kbn-xsrf', 'xxxx')
+    .send(getNewPackagePolicy(packageInfo))
+    .expect(200)
+    .then<PackagePolicy>((response) => response.body.item);
+};
+
+const getPackageInfo = (supertest: SuperTest<Test>) =>
+  supertest
+    .get(`/api/fleet/epm/packages/${CLOUD_SECURITY_POSTURE_PACKAGE_NAME}`)
+    .set('kbn-xsrf', 'xxxx')
+    .expect(200)
+    .then<PackageInfo>((response) => response.body.item);
+
+const getCspRulesSO = (supertest: SuperTest<Test>) =>
+  supertest
+    .get(`/api/saved_objects/_find?type=${CSP_RULE_SAVED_OBJECT_TYPE}`)
+    .expect(200)
+    .then<SavedObjectsFindResponse<CspRule>>((response) => response.body);
 
 export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
-  const chance = new Chance();
-  const kibanaServer = getService('kibanaServer');
-
-  // Failing: See https://github.com/elastic/kibana/issues/139683
-  describe.skip('POST /internal/cloud_security_posture/update_rules_config', () => {
-    let agentPolicyId: string;
 
+  describe('POST /internal/cloud_security_posture/update_rules_config', () => {
     before(async () => {
-      await kibanaServer.savedObjects.cleanStandardList();
       await esArchiver.load('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
-
-      const { body: agentPolicyResponse } = await supertest
-        .post(`/api/fleet/agent_policies`)
-        .set('kbn-xsrf', 'xxxx')
-        .send({
-          name: 'Test policy',
-          namespace: 'default',
-        });
-      agentPolicyId = agentPolicyResponse.item.id;
+      await esArchiver.load('x-pack/test/functional/es_archives/fleet/agents');
```

Comment:
this loads an agent policy with an id of `policy2` which we use in our package policy when creating an integration. 

(also loads some other stuff we don't use)

Reply:
also looks like we should be using `kibanaServer.importExport` but `x-pack/test/functional/es_archives/fleet/agents` hasn't been migrated yet to use `kbnArchiver`